store/src/java/com/zimbra/cs/redolog/op/CreateVolume.java

```diff
@@ -134,4 +147,29 @@ public void redo() throws Exception {
         }
     }
 
+    private VolumeExternalInfo buildVolumeExternalInfo(Volume volume) {
+        VolumeExternalInfo volExtInfo = null;
```

Comment:
This should not be part of this `...redo.CreateVolume` class, right? move this respective file like ExternalVolumeInfoHandler ..

Reply:
Done.